### ISSUE TITLE
Zigbee thermostat heat minimum value at 79

### ISSUE THREAD

**Issue by JSkier21** [mozilla-iot/zigbee-adapter#176]
I have a Centralite HA zigbee thermostat, that works fine when locality is set to Celsius. Settings to Fahrenheit and it enforces a 79 degree (f) minimum in the UI. It will force this temperature until I unpair the unit. 

Running version 0.10.0 on a pi3, using deConz stick.

**Comment by JSkier21** [comment]
Sure thing, see attached. 
 
[zb-00212effff03b05a.json.tar.gz](https://github.com/mozilla-iot/zigbee-adapter/files/3918523/zb-00212effff03b05a.json.tar.gz)

**Comment by mrstegeman** [comment]
This should be fixed in v0.10.4 of the Zigbee add-on.

**Comment by JSkier21** [comment]
It's still really buggy for 0.10.4. Not sure why, but at first it was saying can't set above 65. Then I re-added the device and it set the max value to 71 (what I was trying to set it to before).

**Comment by mrstegeman** [comment]
Yes, that makes sense. Your thermostat enforces a "deadband" of 2.5 °C, which means that the heat target needs to be 2.5 °C lower than the cool target. Right now, you have the cool target set to 75 °F, which means that the maximum heat target is 71 °F.

tl;dr: you need to increase your cooling target before you can increase your heating target.

**Comment by JSkier21** [comment]
Gotcha, this is on a radiator system only, so I didn't check that. 

That said, I can't set cool below 80.42 F. Perhaps this is related to the new Fahrenheit measurement support? Sorry my country has to be difficult.

**Comment by mrstegeman** [comment]
My setup is similar. I only have heat, but my thermostat still has settings for both heat and cool.

You'll need to _raise_ your cooling target, not _lower_ it, in order to have a higher heat target.
